spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinProperties.java

```diff
@@ -27,6 +27,7 @@
 	private String baseUrl = "http://localhost:9411/";
 	private boolean enabled = true;
 	private int flushInterval = 1;
+	private boolean gzip = false;
```

Comment:
Theoretically there can be different types of compressions, right? Maybe it should be an enum compression that defaults to NONE, but can be equal to GZIP. The question is if we support via Mvc any other types of compression - I don't really know.